core/src/main/java/com/netease/arctic/catalog/MixedTables.java

```diff
@@ -18,23 +18,13 @@
 
 package com.netease.arctic.catalog;
 
-import com.netease.arctic.ams.api.TableMeta;
-import com.netease.arctic.ams.api.properties.MetaTableProperties;
+import com.netease.arctic.api.TableMeta;
 import com.netease.arctic.io.ArcticFileIO;
 import com.netease.arctic.io.ArcticFileIOs;
 import com.netease.arctic.io.TableTrashManagers;
-import com.netease.arctic.table.ArcticTable;
-import com.netease.arctic.table.BaseTable;
-import com.netease.arctic.table.BasicKeyedTable;
-import com.netease.arctic.table.BasicUnkeyedTable;
-import com.netease.arctic.table.ChangeTable;
-import com.netease.arctic.table.KeyedTable;
-import com.netease.arctic.table.PrimaryKeySpec;
-import com.netease.arctic.table.TableIdentifier;
-import com.netease.arctic.table.TableMetaStore;
-import com.netease.arctic.table.TableProperties;
-import com.netease.arctic.table.UnkeyedTable;
-import com.netease.arctic.utils.CatalogUtil;
+import com.netease.arctic.properties.MetaTableProperties;
+import com.netease.arctic.table.*;
```

Comment:
We should try to avoid importing the entire package whenever possible.

Reply:
done